.NET/Microsoft.Recognizers.Text.DataTypes.TimexExpression/TimexRegex.cs

```diff
@@ -52,14 +53,16 @@ public static class TimexRegex
                 PeriodCollectionName, new Regex[]
                 {
                     new Regex(@"^P(?<amount>\d*\.?\d+)(?<dateUnit>Y|M|W|D)$"),
-                    new Regex(@"^PT(?<amount>\d*\.?\d+)(?<timeUnit>H|M|S)$"),
+                    new Regex(@"^PT(?<hourAmount>\d*\.?\d+)H(\d*\.?\d+(M|S)){0,2}$"),
```

Comment:
Do we need these to be separate regexes? My worry here is performance.
If combining would make readability much worse, don't do it.